src/components/Profile/ProfileContainer.js

```diff
@@ -41,15 +36,15 @@ const mapStateToProps = (state) => ({
 const WithUrlDataContainerComponent = withRouter(ProfileContainer);
 
 ProfileContainer.defaultProps = {
-  match: PropTypes.func,
+  match: PropTypes.instanceOf(Object),
```

Comment:
See the comments above.

Reply:
I would prefer to do this upon completion of priority tasks